packages/terra-markdown/src/terra-dev-site/doc/markdown/Markdown.doc.jsx

```diff
@@ -4,6 +4,7 @@ import Markdown from 'terra-markdown/lib/Markdown';
 import classNames from 'classnames/bind';
 import ReadMe from '../../../../docs/README.md';
 import styles from './Markdown.module.scss';
+import '../../../../package.json';
```

Comment:
What is this import for?

Reply:
The current setup in the PR is having webpack run through all of the loaded package.json files and generate the respective package markdown files. In Markdown (and Responsive Element), the package.json was never imported anywhere, so the loader never received it from those packages.

Basically, this is a dummy import to make it visible to the loader.